Api driven logan client updates (#1370)
tinacms/tinacms.org: content/docs/features/data-fetching.md

## Making requests with the Tina Client

The Tina client is the easiest way to fetch your site's content. The client can be configured the `.tina/schema.<js|ts>` in the `defineSchema` function.

> Note: token, clientID and branch are **not** used in [local-mode](/docs/tina-cloud/#local-mode). To setup these values for production see [this doc](/docs/tina-cloud/connecting-site)

<!-- TODO: update above doc -->

```js
const schema = defineSchema({
  config: {
    token: '***',
    clientID: '***',
    branch: 'main',
  },
  collections: [
    //...
    // See https://tina.io/docs/schema/ for more info about "collections"
  ],
})

export default schema
```

> When working locally the client will be build with the local url (http://localhost:40001/graphql). When in [production mode]() `clientID`, `branch` and `token` are used to query Tina Cloud.

Reviewer: ```suggestion
When working locally, the client is built with the local url (http://localhost:40001/graphql). When in [production mode]() `clientID`, `branch` and `token` are used to query Tina Cloud.
```
also production mode link is blank here
